CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog]
 
 ## [Unreleased]
 
+- Remove the default maintenance mode availability message
+- Remove the global maintenance mode, now that we can put individual services
+  into maintenance mode
+- A service operator can put an individual service into maintenance mode from
+  the admin interface
```

Comment:
Can we reduce this to one entry to keep things short and sweet.

Reply:
Sure thing.